scripts/esm/first-person-controller.mjs

```diff
@@ -942,7 +955,7 @@ class FirstPersonController extends Script {
      * @range [0, 0.4]
      */
     set gamePadDeadZoneLow(value) {
-        this._gamePadDeadZoneLow = value;
+        this._gamePadDeadZoneLow = value ?? this._gamePadDeadZoneLow;
         if (this._gamePadInput) {
             this._gamePadInput.deadZoneLow = value;
```

Comment:
Since we're checking for nullish, should this not be `this._gamePadInput.deadZoneLow
 = this._gamePadDeadZoneLow`